From 3960bf64fefdf6c66b6738bc2f6288affba13912 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Fri, 13 Jan 2017 14:14:04 -0600
Subject: [PATCH 1474/5242] MLK-13755: ARM: dts: fix the QSPI iomux issue

commit  9bd74e26cba2efa466bef41c5f12b6d89038f928 from
https://source.codeaurora.org/external/imx/linux-imx.git

The iomux PAD setting for QSPI on i.MX7ULP should belong to
iomuxc0(refers to iomuxc in dtsi file) rather than iomuxc1.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-qspi.dts |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-qspi.dts b/arch/arm/boot/dts/imx7ulp-evk-qspi.dts
index 93dbb07..bc9db44 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-qspi.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-qspi.dts
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ * Copyright 2017 NXP
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -22,7 +23,11 @@
 	};
 };
 
-&iomuxc1 {
+&iomuxc {
+	status = "okay";
+};
+
+&iomuxc {
 	imx7ulp-evk {
 		pinctrl_qspi1_1: qspi1grp_1 {
 			fsl,pins = <
-- 
1.7.9.5

